Snippets/Testing/Testing_6/Testing_6.csproj

```diff
@@ -7,5 +7,7 @@
     <PackageReference Include="NServiceBus.Gateway" Version="2.0.*" />
     <PackageReference Include="NServiceBus.Testing" Version="6.*" />
     <PackageReference Include="NUnit" Version="3.*" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.*" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.*" />
```

Comment:
Can you add this to the top of the `ItemGroup` so the package names are sorted properly?